package/docker/Dockerfile

```diff
@@ -11,3 +13,16 @@ COPY package/docker/style.css /usr/local/apache2/htdocs/
 
 # Copy BahmniApps
 COPY ui/dist/. /usr/local/apache2/htdocs/bahmni/
+
+# Download Person Management App
+RUN <<EOR
+    LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/Bahmni/person-management-app/releases/latest)
```

Comment:
Sounds fair Gurpreet. We went with the latest version since we had some development going on actively in this repo and wanted to reflect the changes immediately.

Apologies for the late response :)